process-engine/bpmn-studio
# ✨ Add Feedback Modal

## Changes

1. Add Feedback Modal

PR: #1892

## How to test the changes

- Open the Feedback Modal (Electron menubar > Help > Feedback)
- Enter some information
- Click on the create button
- Save the zip
- Extract the zip
- **Notice that every information provided is included in the created zip.**

> if you know a better place for the feedback button, please let me know.

<img width="202" src="https://user-images.githubusercontent.com/20394992/70329295-5f589780-183b-11ea-9373-740903547a69.png">
<img width="1412" src="https://user-images.githubusercontent.com/20394992/70329294-5f589780-183b-11ea-9d7c-59e656659291.png">
